buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -105,7 +105,7 @@
       },
       "inputs": {
         "filename": "$(Build.SourcesDirectory)\\corefx\\sync.cmd",
-        "arguments": "$(PB_SyncArguments) $(PB_OptionalToolingSyncArguments)",
+        "arguments": "$(PB_SyncArguments) $(PB_OptionalToolingSyncArguments) $(PB_PipelineBuildMSBuildArguments)",
```

Comment:
I assume you know that this works but I'm a little confused as to why it sometimes just keeps `$(variable)` if that value isn't set anywhere. Given you are only setting it in the one case why does this not happen here for the other definitions.